### PR TITLE
Add a README for help content tests

### PR DESCRIPTION
**Pull request by jscook2345**
Attn: @ArangoGutierrez

### CODE REVIEW

File: e2e/help/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+Was the failure a result of an unintended change? If so, we open an issue.
+
+Was the failure a result of an intended change? If so, we update the help text.
+```
+
+* Update the help text in the `testdata/help` directory.
+
+```
+singlularity help inspect > e2e/testdata/help/help-inspect.txt
```
Comment (jchavezlavalle) [inline e2e/help/README.md]: typo here in `singularity`

File: e2e/help/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+## Running the help tests
+
+* To verify this test, modify the `Makefile` to add both a verbose flag and a filter flag (`go test -v -r helpContentTests`) and than run the tests.
```
Comment (jchavezlavalle) [inline e2e/help/README.md]: and **then** run the tests

File: e2e/help/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+* Add the help command to the `helpContentTests` struct in `help.go`
+
+```
+var helpContentTests = []struct {
+        cmds []string
+}{
+	...
+	// singluarity inspect
```
Comment (jchavezlavalle) [inline e2e/help/README.md]: typo here in `singularity inspect` comment

File: e2e/help/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+This package contains the end-to-end tests for `singularity help`. 
+
+## Contributing new help tests
+
+For this example, we're going to create a new test for `singularity help inspect`.
+
+* Add the help text to the `testdata/help` directory.
+
+```
+singlularity help inspect > e2e/testdata/help/help-inspect.txt
```
Comment (jchavezlavalle) [inline e2e/help/README.md]: typo in `singularity`

**Review by jchavezlavalle — DISMISSED** [review]

**Comment by jscook2345** [comment]
Fixed typos! Thanks @devprincess !

File: e2e/help/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# Testing `singularity help` content
```
Comment (ArangoGutierrez) [inline e2e/help/README.md:1]:
```suggestion
# Testing singularity help content
```

File: e2e/help/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+* Add the help text to the `testdata/help` directory.
+
+```
+singularity help inspect > e2e/testdata/help/help-inspect.txt
+```
+
+* Add the help command to the `helpContentTests` struct in `help.go`
+
+```
```
Comment (ArangoGutierrez) [inline e2e/help/README.md:17]:
```suggestion
```go
```

File: e2e/help/README.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+Was the failure a result of an unintended change? If so, we open an issue.
+
+Was the failure a result of an intended change? If so, we update the help text.
+```
+
+* Update the help text in the `testdata/help` directory.
+
+```
```
Comment (ArangoGutierrez) [inline e2e/help/README.md:42]:
```suggestion
```bash
```

File: e2e/help/README.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+singularity help inspect > e2e/testdata/help/help-inspect.txt
+```
+
+## Running the help tests
+
+* To verify this test, modify the `Makefile` to add both a verbose flag and a filter flag (`go test -v -r helpContentTests`) and then run the tests.
+
+```
```
Comment (ArangoGutierrez) [inline e2e/help/README.md:50]:
```suggestion
```bash
```

File: e2e/help/README.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+This package contains the end-to-end tests for `singularity help`. 
+
+## Contributing new help tests
+
+For this example, we're going to create a new test for `singularity help inspect`.
+
+* Add the help text to the `testdata/help` directory.
+
+```
```
Comment (ArangoGutierrez) [inline e2e/help/README.md:11]:
```suggestion
```bash
```

**Review by ArangoGutierrez — APPROVED** [review]

**Pull request merged**